frontend/src/components/Footer.tsx

```diff
@@ -1,10 +1,10 @@
 import { useRecoilValue } from 'recoil';
 import github from '../assets/github.png';
-import { Link } from "react-router-dom";
+import { Link } from 'react-router-dom';
 import { loggedInState } from '../store/atoms/auth';
 import { CgProfile } from 'react-icons/cg';
-import { BsFilePost } from 'react-icons/bs';
-import { FaHome,FaRegCopyright } from 'react-icons/fa';

```

Comment:
Story was related to Navbar Icons, why we need changes in Footer.tsx ?
If scope is changes, could you update in Issue ?